analyzer/src/main/kotlin/AnalyzerResultBuilder.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
```

Comment:
Please keep the copyright holder: 2017-2019 HERE Europe B.V.

Reply:
Done.